validate-differential-expression/gene_DE_validate.Rmd

```diff
@@ -114,8 +124,9 @@ Note that these differential expression results come with Affymetrix probe IDs
 
 ```{r}
 # Load in the summary statistics from limma run
-author.full.results <- readr::read_tsv(file.path("results", 
-                                                 "GSE37382_limma_results.tsv")) 
+author.full.results <- readr::read_tsv(file.path(results_dir, 
+                                                 "GSE37382_limma_results.tsv" # Replace with name of `limma` results file generated in `author_processed_DE.Rmd`
```

Comment:
```suggestion
                                                 "GSE37382_limma_results.tsv" # Replace with name of your own differential expression results file you've generated that you'd like to compare to refinebio data
```
We should make it more clear that it doesn't necessarily need to be this file. The idea of this example is that people can provide their own limma results file to compare with refinebio results. (the `author_processed_DE.Rmd` notebook is just a set up notebook that we used as an example of a file you could compare to, but ideally the user would have some sort of file like this already themselves). 

This being said, we should add more comments at places where metadata or column names for `author.full.results` might differ when our user uploads their own file. Just so they have more heads up where things are likely not to work for every file. (We can't guarantee their limma results file will look the same as ours).